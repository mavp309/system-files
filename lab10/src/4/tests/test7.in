1000 2000 300000 400000 50000
100
