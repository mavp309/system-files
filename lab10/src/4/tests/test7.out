1000000002000000300000000400000000050100
